misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/IterableAssertionsSpec.kt

```diff
@@ -40,6 +43,8 @@ abstract class IterableAssertionsSpec(
     val hasDescriptionBasic = DescriptionBasic.HAS.getDefault()
     val hasNotDescriptionBasic = DescriptionBasic.HAS_NOT.getDefault()
     val nextElement = DescriptionIterableAssertion.NEXT_ELEMENT.getDefault()
+    val duplicateElements = DescriptionIterableAssertion.DUPLICATE_ELEMENTS.getDefault()
+    val indexElements = DescriptionIterableAssertion.INDEX.getDefault()
```

Comment:
can be removed once you use the other suggestion
```suggestion
```

Reply:
ok